Makefile.in

```diff
@@ -549,13 +551,13 @@ install: default fltools
 	rm -f $(DESTDIR)$(bindir)/spud-* $(DESTDIR)$(bindir)/diamond $(DESTDIR)$(bindir)/silenteval $(DESTDIR)$(bindir)/runut
 	[ -f manual/fluidity_manual.pdf ] && cp manual/fluidity_manual.pdf $(DESTDIR)$(docdir)/fluidity || true
 	mkdir -p $(DESTDIR)$(datadir)/fluidity
-	cp -r schemas/ $(DESTDIR)$(datadir)/fluidity/
```

Comment:
This one interests me - I'd have expected that the trailing / would be wanted rather than not wanted, specifying the copy of the directory itself. Out of interest what's the issue with having it in place there? (Is this sometimes a link, perhaps?)